src/test/java/core/basesyntax/JavaDateTimeApiTest.java

```diff
@@ -2,8 +2,7 @@
 
 import org.junit.Assert;
 import org.junit.Test;
-
-import java.time.LocalDate;
+import java.time.*;
```

Comment:
it's bad practice. 
let's return to the previous state

Reply:
Sorry ! It was automatically